packer/sagebio/sagebio-base-ubuntu-bionic-LATEST/playbook.yaml

```diff
@@ -18,10 +18,11 @@
         state: present
 
     # awscli is on aws linux2 distro by default
-    - name: install awscli
+    - name: install awscli and SSM
```

Comment:
maybe should rename this to "install snap packages" ?

Reply:
Agree!